turbo.json

```diff
@@ -49,123 +67,49 @@
+      "cache": false,
+      "persistent": true,
+      "outputs": [".stylelintcache"]
     },
     "lint": {
-      "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx}",
-        "{app,components,layouts,pages,styles}/**/*.css",
-        "{next-data,scripts,i18n}/**/*.mjs",
-        "{.storybook,public}/**/*.{ts,js,css}",
-        "{app,pages}/**/*.{mdx,md}",
-        "*.{md,mdx,ts,tsx,mjs,yml}",
-        ".eslintrc.json",
-        ".eslintignore",
-        ".stylelintrc.mjs",
-        ".stylelintignore",
-        ".prettierrc.json",
-        ".prettierignore",
-        ".remarkrc.json"
-      ],
-      "outputs": [".eslintjscache", ".eslintmdcache"]
+      "cache": false,
+      "persistent": true,
+      "outputs": [".eslintjscache", ".eslintmdcache", ".stylelintcache"]
     },
     "lint:fix": {
-      "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx}",
-        "{app,components,layouts,pages,styles}/**/*.css",
-        "{next-data,scripts,i18n}/**/*.mjs",
-        "{.storybook,public}/**/*.{ts,js,css}",
-        "{app,pages}/**/*.{mdx,md}",
-        "*.{md,mdx,ts,tsx,mjs}",
-        ".eslintrc.json",
-        ".eslintignore",
-        ".stylelintrc.mjs",
-        ".stylelintignore",
-        ".prettierrc.json",
-        ".prettierignore",
-        ".remarkrc.json"
-      ],
-      "outputs": [".eslintjscache", ".eslintmdcache"]
+      "cache": false,
+      "persistent": true,
+      "outputs": [".eslintjscache", ".eslintmdcache", ".stylelintcache"]
     },
     "prettier": {
-      "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx}",
-        "{app,components,layouts,pages,styles}/**/*.css",
-        "{next-data,scripts,i18n}/**/*.{mjs,json}",
-        "{.storybook,public}/**/*.{ts,js,css,json}",
-        "{app,pages}/**/*.{mdx,md}",
-        "*.{md,mdx,json,ts,tsx,mjs,yml}",
-        ".eslintrc.json",
-        ".eslintignore",
-        ".stylelintrc.mjs",
-        ".stylelintignore",
-        ".prettierrc.json",
-        ".prettierignore",
-        ".remarkrc.json",
-        ".lintstaged.json"
-      ],
+      "cache": false,
+      "persistent": true,
       "outputs": [".prettiercache"]
     },
     "prettier:fix": {
-      "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx}",
-        "{app,components,layouts,pages,styles}/**/*.css",
-        "{next-data,scripts,i18n}/**/*.{mjs,json}",
-        "{.storybook,public}/**/*.{ts,js,css,json}",
-        "{app,pages}/**/*.{mdx,md}",
-        "*.{md,mdx,json,ts,tsx,mjs,yml}",
-        ".eslintrc.json",
-        ".eslintignore",
-        ".stylelintrc.mjs",
-        ".stylelintignore",
-        ".prettierrc.json",
-        ".prettierignore",
-        ".remarkrc.json",
-        ".lintstaged.json"
-      ],
+      "cache": false,
+      "persistent": true,
```

Comment:
I don't think `persistent` necessary here and in other `:fix` scenarios. It was my understanding that `persistent` was used for commands that watch by default and these commands do not.

https://turbo.build/repo/docs/reference/configuration#persistent

Reply:
Oh! TIL I learned that! Will make proper updates!